workers/unity/Assets/Playground/Scripts/Cubes/CollisionProcessSystem.cs

```diff
@@ -0,0 +1,76 @@
+using Generated.Playground;
+using Improbable.Gdk.Core;
+using Unity.Collections;
+using Unity.Entities;
+
+namespace Playground
+{
+    [RemoveAtEndOfTick]
+    public struct CollisionComponent : IComponentData
+    {
+        public Entity ownEntity;
+        public Entity otherEntity;
+    }
+
+    [UpdateInGroup(typeof(SpatialOSUpdateGroup))]
+    internal class CollisionProcessSystem : ComponentSystem
+    {
+        private struct Data
+        {
+            public readonly int Length;
+            public ComponentDataArray<CollisionComponent> Collision;
+            public EntityArray Entities;
+            public ComponentDataArray<SpatialOSLaunchable> Launchable;
```

Comment:
nit: personally prefer to have them ordered by `readonly`, type of array, .. etc.
so don't have a `ComponentDataArray`, `EntityArray `, `ComponentDataArray`. makes it a bit harder to read

Reply:
We don't seem to have an agreed way of ordering them. This seemed OK me, as `CollisionComponent` objects are the only ones that the collision monobehaviour injects.